lib/util/cmcd_manager.js

```diff
@@ -841,3 +872,10 @@ shaka.util.CmcdManager.StreamingFormat = {
  * @const {number}
  */
 shaka.util.CmcdManager.Version = 1;
+
+
+/**
+ * @const {number}
+ * @private
+ */
+shaka.util.CmcdManager.RTP_SAFETY_FACTOR_ = 5;
```

Comment:
This should be configurable. Can we add it to `shaka.extern.CmcdConfiguration`?

Reply:
ok, I'll add it

Reply:
Done!